src/Calendar/CalendarFactory.php

```diff
@@ -209,8 +210,8 @@ private function createOpeningHoursFromWeekScheme(
                     );
 
                     $openingHour = new OpeningHour(
-                        OpeningTime::fromNativeDateTime($opens),
-                        $closes ? OpeningTime::fromNativeDateTime($closes) : OpeningTime::fromNativeDateTime($opens),
+                        Time::fromDateTime($opens),
+                        $closes ? Time::fromDateTime($closes) : Time::fromDateTime($opens),
```

Comment:
```suggestion
                        Time::fromDateTime($closes ?? $opens),
```

Reply:
Ah, but this will not work, according to PHPStan "Variable $closes on left side of ?? always exists and is not nullable."